### PR TITLE
Add ansible deployment

### PR DESCRIPTION
**Pull request by KTachibanaM**
This PR adds instructions on how to use [an Ansible playbook](https://github.com/k-t-corp/RSSHub-ansible) that I wrote to deploy RSSHub on bare-metal with Redis, browserless and Caddy 2

## 该 PR 相关 Issue / Involved issue

Close #

### CODE REVIEW

**Comment by NeverBehave** [comment]
建议将相关代码放置在项目`scripts/ansible`目录而不是引用到外部仓库, 方便后期维护

**Comment by KTachibanaM** [comment]
@NeverBehave makes sense. will do

File: docs/install/README.md [hunk 1]
```diff
@@ -106,6 +106,35 @@ $ docker run -d --name rsshub -p 1200:1200 -e CACHE_EXPIRE=3600 -e GITHUB_ACCESS
 
 更多配置项请看 [#配置](#pei-zhi)
 
+## Ansible 部署
+
+这个 Ansible playbook 包括了 RSSHub, Redis, browserless 以及 Caddy 2
+
+目前只支持 Ubuntu 20.04
+
+需要 sudo 权限
```
Comment (NeverBehave) [inline docs/install/README.md:115]: 建议注明依赖`Docker`, `OpenVZ 7` 这一类的机器也将无法使用

实际上, 如果可以给一个选项是否激活`browserless`会比较好: 大部分路由都不需要它, 可以降低门槛

Comment (KTachibanaM) [reply docs/install/README.md:115]: I can mention Docker, but I don't think it makes sense to exclude browserless for Ansible deployment? Ansible is usually used for servers where users have access to the OS (and hence virtualization capability like Docker), for example a DigitalOcean or Linode VPS, so I'd expect Ansible deployment to install as full-featured as possible.
Excluding browserless to preserve resources is another argument, and I can make it another PR.

**Review by NeverBehave — APPROVED** [review]
LGTM

**Pull request merged**